dev-requirements.txt

```diff
@@ -10,12 +10,13 @@ async-timeout==3.0.1
 asynctest==0.13.0
 atomicwrites==1.3.0       # via pytest
 attrs==19.1.0
+azure_storage==0.36.0
```

Comment:
Why was this pulled in? There's no direct dependency in `dev-requirements.in` and it isn't pulled in via `__app__/requirements.txt`.

Reply:
Sorry about 'azure_storage' and other issues, I misunderstood a few things. I wanted to clarify if upgrading to aiohttp to 3.6.2 (from 3.5.4) would break anything. I will create a new PR with changes. Thank you for your time.